### PR TITLE
Added release docs for 1.18-2

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*
N/A

*Description of changes:*
Feedback is always welcome and appreciated, but this is a WIP/hold until we're closer to releasing 1.18.2. 

Added index.md and CHANGELOG-v1-18-eks-2.md for 1.18.2. SNS notification body (release-announcement.txt) will be a separate and later PR, as it will trigger the SNS.

Although not ideal, CHANGELOG-v1-19-eks-1.md  was also updated, even though this is for 1-19 and is already out. The patch added for the 1.18.2 release will not be a patch for 1.19 or for any other release in the future. Its removal had to be noted somewhere, and including it in the changelog for 1.19.1 with the other patches that were removed for that release seemed like the best solution

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/releases/1-18/2/index.md [hunk 1]
```diff
@@ -4,6 +4,654 @@ The [EKS-D v1-18-eks-2](https://distro.eks.amazonaws.com/kubernetes-1-18/kuberne
 release manifest defines the release used in EKS-D.
 
 ```yaml
----
-TODO: add crd here
+---apiVersion: distro.eks.amazonaws.com/v1alpha1
```
Comment (TerryHowe) [inline docs/contents/releases/1-18/2/index.md]:
```suggestion
---
apiVersion: distro.eks.amazonaws.com/v1alpha1
```

File: docs/contents/releases/1-18/2/index.md [hunk 2]
```diff
@@ -0,0 +1,657 @@
+# EKS-D v1-18-eks-2 Release
+
+The [EKS-D v1-18-eks-2](https://distro.eks.amazonaws.com/kubernetes-1-18/kubernetes-1-18-eks-2.yaml)
+release manifest defines the release used in EKS-D.
+
+```yaml
+apiVersion: distro.eks.amazonaws.com/v1alpha1
```
Comment (TerryHowe) [inline docs/contents/releases/1-18/2/index.md:7]:
```suggestion
---a
apiVersion: distro.eks.amazonaws.com/v1alpha1
```

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/contents/releases/1-18/2/CHANGELOG-v1-18-eks-2.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    are part of Kubernetes 1.19, so this patch will only apply to 1.18.
+  * The changes in this patch fix a bug related to AWS Network Load Balancer (NLB) and cordoned nodes. Previously, 
+    cordoned nodes were erroneously not deregistered from an NLB's target group, despite being perceived as already out
+    of the target group. Now, with this patch, Cordoned nodes are deregistered from target groups as expected.
+* **0018-EKS-PATCH-Delete-leaked-volume-if-driver-doesn-t-kno.patch** 
+  * This patch applies the changes from upstream Kubernetes [PR #99664](https://github.com/kubernetes/kubernetes/pull/99664),
+    which is open as of the time of EKS-D v1.18.2 release, and is raised in [issue #754](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/754) 
+    in aws-ebs-csi-driver.
+  * The changes in this patch fix a bug where a volume could be created during the create process, but the driver does 
+    not know the status of this volume. The changes in this patch now delete the leaked driver.
```
Comment (taneyland) [inline docs/contents/releases/1-18/2/CHANGELOG-v1-18-eks-2.md:23]: From the title of this patch, should the end of this line say this instead?
```suggestion
    not know the status of this volume. The changes in this patch now delete the leaked volume.
```

**Pull request merged**